app/presenters/tree_builder_infra_networking.rb

```diff
@@ -16,7 +16,7 @@ def tree_init_options(_)
 
   def set_locals_for_render
     locals = super
-    locals.merge!(:id_prefix => 'nt_', :autoload => true)
+    locals.merge!(:autoload => true)
```

Comment:
fixed :)

I'll probably do a separate PR to unify all the various ways this function is written to something that just returns a hash..